libgraph/include/katana/analytics/k_shortest_paths/sssp.h

```diff
@@ -0,0 +1,24 @@
+#ifndef KATANA_LIBGRAPH_KATANA_ANALYTICS_KSHORTESTPATHS_SSSP_H_
+#define KATANA_LIBGRAPH_KATANA_ANALYTICS_KSHORTESTPATHS_SSSP_H_
+
+#include "katana/analytics/sssp/sssp.h"
+
+namespace katana::analytics {
+enum AlgoReachability { async = 0, syncLevel };
+
+/// Compute the Single-Source Shortest Path for pg starting from start_node.
+/// The edge weights are taken from the property named
+/// edge_weight_property_name (which may be a 32- or 64-bit sign or unsigned
+/// int), and the computed path lengths are stored in the property named
+/// output_property_name (as uint32_t). The algorithm and delta stepping
+/// parameter can be specified, but have reasonable defaults.
+/// The property named output_property_name is created by this function and may
+/// not exist before the call.
```

Comment:
update the above documentation to ksssp

Reply:
Let me know if more comments need to be added

Reply:
this is fine